Solutions/Recorded Future/Playbooks/readme.md

```diff
@@ -1,11 +1,28 @@
 # RecordedFuture - Import Indicators Logic Apps and IOC Enrichment Playbook templates
-# Overview
+<img alt="Recorded Future" src="https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/RecordedFuture.svg" with=80 height=80/>
 
+# Overview
 Recorded Future is the world’s largest provider of intelligence for enterprise security. By combining persistent and pervasive automated data collection and analytics with human analysis, Recorded Future delivers intelligence that is timely, accurate, and actionable.
 
+# Roles and permissions for working with Recorded Future Solution
+
+The following article describes roles and permissions for working with Microsoft Sentinel [https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#microsoft-sentinel-contributor](https://learn.microsoft.com/en-us/azure/sentinel/roles).
+
+To install and manage Playbooks/Logic Apps, the followong permissions are required on the resourcegroup [Microsoft Sentinel Contributor](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#microsoft-sentinel-contributor) + [Logic App Contributor](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#logic-app-contributor).
+
+# Connectors Authorization 
+Each connector needs to be authorized after playbook installation. Expand all nodes in the logic app after installation and look for blocks marked with a warning sign. Open and authorize the connection. More information about each connector can be found below. 
+
+We use the following connectors in Recordedd Future solution:
+- **/recordedfuturev2** Microsoft power platform connectors 
+https://learn.microsoft.com/en-us/connectors/recordedfuturev2/  . All logic apps require an APIKey to communicate with the Recorded Future API. To obtain an APIKey, please visit [Recorded Future Requesting API Tokens](https://support.recordedfuture.com/hc/en-us/articles/4411077373587-Requesting-API-Tokens).
+- **/microsoftgraphsecurity** - Microsoft power platform connectors https://learn.microsoft.com/en-us/connectors/microsoftgraphsecurity/
+- **/azuresentinel** - Microsoft power platform connectors https://learn.microsoft.com/en-us/connectors/azuresentinel/
+
+
 # Deployment
 
-> **Due to internal Microsoft Logic Apps dependencies, please deploy first the ImportToSentinel playbook before any of the IndicatorProcessor one.**
+> **Due to internal Microsoft Logic Apps dependencies, please deploy ImportToSentinel playbook before any of the IndicatorProcessor playbooks.**
 
 ## ImportToSentinel
 
```

Comment:
In the next line, 
1. Please correct the spelling of "optimizition "
2. Instead of "This Playbooks" it should be "This Playbook" or "These Playbooks"